docs/quickoverview.rst

```diff
@@ -0,0 +1,46 @@
+***********************
+A Quick Overview
+***********************
+
+Gumby is an experiment runner framework to run local and distributed experiments.
+
+
+The current supported DAS environments are the DAS5 and DAS6 clusters.
+More information on the DAS environments can be found at:
+`DAS5  <https://www.cs.vu.nl/das5/clusters.shtml>`_.
+`DAS6  <https://www.cs.vu.nl/das6/clusters.shtml>`_.
+
+The interaction between the researcher and the gumby framework can be presented als follows:
+
+.. image:: MdVGumbyOverview.png
+   :width: 600
+
+
+Installation
+
+Prior to installing Gumby install the required dependencies for basic tests.
+Those are the software packages: 
+python-psutil
```

Comment:
I don't think it's necessary to explicitly list the required packages since they are already part of the `apt-get install` command. The reader would know the requirements by looking at the command below.